get_started/what_is_citus.rst

```diff
@@ -15,7 +15,7 @@ SaaS applications, customer-facing real-time analytics dashboards, and time
 series workloads.
 
 Two Ways to Get Citus:
-$$$$$$$$$$$$$$$$$$$$$$
+^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This still doesn't build.

```
reading sources... [100%] use_cases/timeseries


Warning, treated as error:
/home/awolk/citus_docs/get_started/what_is_citus.rst:64:Title level inconsistent:

Multi-Tenant SaaS Database
--------------------------
make: *** [Makefile:4: docs] Error 2
```

Like I said before, the character are not assigned to specific H levels. You can't assume that `^` is H3.

By using the following order:

![image](https://github.com/user-attachments/assets/7218d195-df80-4bed-8717-78e1359c7189)

```
level 1
====

level 2
^^^^

level 3
-------
```
You make it expect that setup in all further sections, hence when on line 64 it sees `-----` instead of `^^^^^` it reports inconsistent leveling.


![image](https://github.com/user-attachments/assets/f46059df-8433-4a9a-acac-5fe4cc15d750)

Reply:
I see. We'll have to contest this bug because "Two Ways to Get Citus" doesn't warrant an h2.